packages/api/migrations/20201013014125-nomination_columns.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.removeColumn('nominations', 'providerFirstName'),
+      queryInterface.removeColumn('nominations', 'providerLastName'),
+      queryInterface.removeColumn('nominations', 'relationship'),
+      queryInterface.removeColumn('nominations', 'patientFirstName'),
+      queryInterface.removeColumn('nominations', 'patientLastName'),
+      queryInterface.removeColumn('nominations', 'age'),
+
+      queryInterface.addColumn('nominations', 'providerName', {
+        type: Sequelize.STRING,
+      }),
+      queryInterface.addColumn('nominations', 'patientName', {
+        type: Sequelize.STRING,
+      }),
+      queryInterface.addColumn('nominations', 'representativeRelationship', {
+        type: Sequelize.STRING,
+      }),
+      queryInterface.addColumn('nominations', 'patientAge', {
+        type: Sequelize.INTEGER,
+      }),
+      queryInterface.addColumn('nominations', 'representativePhoneNumber', {
+        type: Sequelize.INTEGER,
```

Comment:
I'd consider using a string here instead. Though phone numbers contain numbers, they're not really integers. It doesn't make sense, for example, to add or divide them.

Reply:
agreed

Reply:
I was considering changing providerNumber to providerPhoneNumber and making a string. 
Am I being too nit picky about the name sake?

Reply:
and changing title to proviederTitle

Reply:
I agree that we should go with `string` ! I think going with `providerPhoneNumber` `providerTitle` makes sense as well since that seems to be the convention we are following. 

Maybe in the future `Provider` becomes its own model 🤔

Reply:
It would definitely be helpful if one provider has many applications. It would also be an easier way to store verified email/provider